### ISSUE TITLE
On feedforward control using physics-guided neural networks: Training cost regularization and optimized initialization. (arXiv:2201.12088v1 [cs.LG])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#15253]
On feedforward control using physics-guided neural networks: Training cost regularization and optimized initialization. (arXiv:2201.12088v1 [cs.LG])<br>
https://ift.tt/DASbdwr1O<br>
<p>Performance of model-based feedforward controllers is typically limited by the accuracy of the inverse system dynamics model. Physics-guided neural networks (PGNN), where a known physical model cooperates in parallel with a neural network, were recently proposed as a method to achieve high accuracy of the identified inverse dynamics. However, the flexible nature of neural networks can create overparameterization when employed in parallel with a physical model, which results in a parameter drift during training. This drift may result in parameters of the physical model not corresponding to their physical values, which increases vulnerability of the PGNN to operating conditions not present in the training data. To address this problem, this paper proposes a regularization method via identified physical parameters, in combination with an optimized training initialization that improves training convergence. The regularized PGNN framework is validated on a real-life industrial linear motor, where it delivers better tracking accuracy and extrapolation.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
